standup/status/jinja2/500.html

```diff
@@ -23,6 +23,15 @@
       <div id="main-content" class="container_12 cf">
         <div class="grid_8 prefix_2 suffix_2">
           <div class="error-message">{{ title }}</div>
+          <div>
```

Comment:
I was able to test this locally, and recommend [this markup](https://gist.github.com/pmac/d36987c7cde0f1c0dda71459e5afa8b1) for asthetics.